diff --git a/src/drivers/net/bnxt/bnxt.h b/src/drivers/net/bnxt/bnxt.h
index 78288196..d42d3246 100644
--- a/src/drivers/net/bnxt/bnxt.h
+++ b/src/drivers/net/bnxt/bnxt.h
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2018 Broadcom. All Rights Reserved. 
+ * Copyright © 2018 Broadcom. All Rights Reserved.
  * The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries.
 
  * This program is free software; you can redistribute it and/or modify it under
@@ -7,7 +7,7 @@
  * Free Software Foundation.
 
  * This program is distributed in the hope that it will be useful.
- * ALL EXPRESS OR IMPLIED CONDITIONS, REPRESENTATIONS AND WARRANTIES, INCLUDING 
+ * ALL EXPRESS OR IMPLIED CONDITIONS, REPRESENTATIONS AND WARRANTIES, INCLUDING
  * ANY IMPLIED WARRANTY OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, OR
  * NON-INFRINGEMENT, ARE DISCLAIMED, EXCEPT TO THE EXTENT THAT SUCH DISCLAIMERS
  * ARE HELD TO BE LEGALLY INVALID. See the GNU General Public License for more
@@ -150,7 +150,7 @@ union dma_addr64_t {
 #define DEFAULT_NUMBER_OF_RX_RINGS              0x01
 #define DEFAULT_NUMBER_OF_RING_GRPS             0x01
 #define DEFAULT_NUMBER_OF_STAT_CTXS             0x01
-#define NUM_RX_BUFFERS                          8
+#define NUM_RX_BUFFERS                          2
 #define MAX_RX_DESC_CNT                         16
 #define MAX_TX_DESC_CNT                         16
 #define MAX_CQ_DESC_CNT                         64
